islandora_compound_object.module

```diff
@@ -270,7 +270,7 @@ function islandora_compound_object_access(AbstractObject $object) {
 /**
  * Implements hook_compound_cmodel_islandora_derivative().
```

Comment:
This should say: "* Implements hook_CMODEL_islandora_derivative()".

Reply:
Done.